tests/lib/rules/space-in-parens.js

```diff
@@ -115,319 +120,417 @@ ruleTester.run("space-in-parens", rule, {
     ],
 
     invalid: [
+
+        // methods and functions
+        {
+            code: "bar(baz )",
+            output: "bar(baz)",
+            options: ["never"],
+            errors: [REJECTED_CLOSING_SPACE_ERROR]
+        },
+        {
+            code: "bar( baz )",
+            output: "bar(baz)",
+            options: ["never"],
+            errors: [
+                { message: REJECTED_OPENING_SPACE_ERROR, line: 1, column: 4 },
+                { message: REJECTED_CLOSING_SPACE_ERROR, line: 1, column: 10 }
+            ]
+        },
         {
             code: "foo( )",
             output: "foo()",
             options: ["never"],
-            errors: [{ message: REJECTED_SPACE_ERROR, line: 1, column: 4 }]
+            errors: [
+                { message: REJECTED_OPENING_SPACE_ERROR, line: 1, column: 4 },
+                { message: REJECTED_CLOSING_SPACE_ERROR, line: 1, column: 6 }
+            ]
+        },
+        {
+            code: "foo(bar() )",
+            output: "foo(bar())",
+            options: ["never"],
+            errors: [REJECTED_CLOSING_SPACE_ERROR]
+        },
+        {
+            code: "foo\n(\nbar )",
+            output: "foo\n(\nbar)", // actual error
```

Comment:
What is the significance of this comment?

Reply:
Seems like a relic I left for myself which slipped through, will remove :zap: